packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -45,24 +45,47 @@ import {
 import { SegmentGroupCollection } from "./segmentGroupCollection";
 import { PropertiesManager, PropertiesRollback } from "./segmentPropertiesManager";
 
+/**
+ * Common properties for a node in a merge tree.
+ */
 export interface IMergeNodeCommon {
     parent?: IMergeBlock;
     /**
      * The length of the contents of the node.
      */
     cachedLength: number;
+    /**
+     * The index of this node in its parent's list of children.
+     */
     index: number;
+    /**
+     * A string that can be used for comparing the location of this node to other `MergeNode`s in the same tree.
+     * `a.ordinal < b.ordinal` if and only if `a` comes before `b` in a pre-order traversal of the MergeTree.
+     */
     ordinal: string;
     isLeaf(): this is ISegment;
 }
 
 export type IMergeNode = IMergeBlock | ISegment;
 
-// Node with segments as children
+/**
+ * Internal node in a merge tree.
+ */
 export interface IMergeBlock extends IMergeNodeCommon {
     needsScour?: boolean;
+    /**
+     * Number of children of this node
```

Comment:
would be good to clarify if this is immediate children, or total subtree size (if so, does it include the current node or not)

Reply:
```suggestion
     * Number of direct children of this node
```

Reply:
From the below, looks like immediate children.
```suggestion
     * Number of immediate children of this node. The members of `children` less than this are used.
```